### PR TITLE
unzip: bump revision for new patches

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Ubuntu hasn't updated for Debian's new security patches, so jumping over to Debian's patches _(mostly)_ instead for now.

### CODE REVIEW

File: Formula/unzip.rb [hunk 1]
```diff
@@ -15,31 +15,41 @@ class Unzip < Formula
...
-      patches/10-cve-2014-8140-test-compr-eb
-      patches/11-cve-2014-8141-getzip64data
-      patches/12-cve-2014-9636-test-compr-eb
-      patches/13-remove-build-date
-      patches/14-cve-2015-7696
-      patches/15-cve-2015-7697
-      patches/16-fix-integer-underflow-csiz-decrypted
-      patches/20-unzip60-alt-iconv-utf8
-    ]
+    apply "patches/20-unzip60-alt-iconv-utf8"
```
Comment (ilovezfs) [inline Formula/unzip.rb]: do we really need this one?

Comment (DomT4) [inline Formula/unzip.rb]: Unsure. The patch description is:

```
From: Giovanni Scafora <giovanni.archlinux.org>
Subject: unzip files encoded with non-latin, non-unicode file names
Last-Update: 2015-02-11

Updated 2015-02-11 by Marc Deslauriers <marc.deslauriers@canonical.com>
to fix buffer overflow in charset_to_intern()
```
Seems useful, but who knows if anyone is actually using that fixed functionality.

Comment (DomT4) [inline Formula/unzip.rb]: Actually `to fix buffer overflow in charset_to_intern()` makes it sound fairly mandatory, since that's a vuln 😓.

Comment (ilovezfs) [inline Formula/unzip.rb]: might want to find out why Debian hasn't adopted it but I guess it can continue to squat

Comment (DomT4) [inline Formula/unzip.rb]: Sure, I'll mail the Debian maintainer to ask.

Comment (DomT4) [inline Formula/unzip.rb]:
```
It fixes a buffer overflow in a previous version of the same patch,
not a buffer overflow in the Debian unzip package.

The patch seems to introduce a new feature regarding charsets. It is
said to "fix" things like this one:

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=197427

But it's very intrusive, and I don't want to deviate from upstream to
add new features, they might be later incompatible with whatever
upstream decides to do. IMHO, new features should be introduced by the
authors, not by Debian, at least for software which has upstream
maintainers.

Thanks.
```

Comment (ilovezfs) [inline Formula/unzip.rb]: Sounds like we should drop it then.

Comment (DomT4) [inline Formula/unzip.rb]: Your wish is my command 😉

**Comment by ilovezfs** [comment]
:zipper_mouth_face: :ship:

**Pull request closed without merging**